### PR TITLE
team 3

### PR DESCRIPTION
**Pull request by Zorni3001**

### CODE REVIEW

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 1]
```diff
@@ -5,37 +5,41 @@
...
+	public List<Long> findEurekaNumbersBetween(Long lowerBound, Long upperBound) {
 
-		List<Long> result = new ArrayList<Long>();
+		var result = new ArrayList<Long>();
 
-		for (long i = a; i < b; i++) {
+		for (var i = lowerBound; i < upperBound; i++) {
 
-			List<Long> longs = new ArrayList<Long>();
+			var numberAsString = Long.toString(i);
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:14]: Dont use var. Use String in this case (also dont use var for Lists)

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 2]
```diff
@@ -5,37 +5,41 @@
...
-				sum += Math.pow(longs.get(j - 1), j);
-			}
-
-			// Test if sum is equal
-			if (sum == i) {
-				result.add(sum);
-			}
+			long eurekaOfValues = calculateEureka(values);
 
+			if (eurekaOfValues == i) result.add(eurekaOfValues);
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:19]: One line Ifs are tricky.

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 3]
```diff
@@ -5,37 +5,41 @@
...
 		}
 
         System.out.println(result);
 		return result;
 
 	}
 
+	private List<Long> splitValues(String input){
+		var ret = new ArrayList<Long>();
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:28]: What is ret?

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 4]
```diff
@@ -5,37 +5,41 @@
...
-			// Test if sum is equal
-			if (sum == i) {
-				result.add(sum);
-			}
+			long eurekaOfValues = calculateEureka(values);
 
+			if (eurekaOfValues == i) result.add(eurekaOfValues);
 		}
 
         System.out.println(result);
```
Comment (SebastianHerzog) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:22]: Einrückung

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 5]
```diff
@@ -5,37 +5,41 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> findEurekaNumbersBetween(Long lowerBound, Long upperBound) {
```
Comment (Bananamafia) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:8]: gute Variablen 👍

File: src/main/java/de/cas/mse/exercise/SumDigPower.java (hunk 3 above)
Comment (Bananamafia) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:28]: splittedValues statt ret? oder result?

**Pull request merged**